util/vars.yml

```diff
@@ -0,0 +1,200 @@
+
+  - lat: 22.0
+    lon: 18.0
+    nz: ~  # all
+    atm: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.atmf012.nc"
+    sfc: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.sfcf012.nc"
+    out: "../tests/MetProfiles/Profile_SAHARA_Aug_12z.csv"
+
+  - lat: 22.0
+    lon: 18.0
+    nz: ~  # all
+    atm: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.atmf006.nc"
+    sfc: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.sfcf006.nc"
+    out: "../tests/MetProfiles/Profile_SAHARA_Aug_06z.csv"
+
+  - lat: -5.0
+    lon: -65.0
+    nz: ~  # all
+    atm: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.atmf012.nc"
+    sfc: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.sfcf012.nc"
+    out: "../tests/MetProfiles/Profile_AMAZON_Aug_12z.csv"
+
+  - lat: -5.0
+    lon: -65
+    nz: ~  # all
+    atm: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.atmf018.nc"
+    sfc: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.sfcf018.nc"
+    out: "../tests/MetProfiles/Profile_AMAZON_Aug_18z.csv"
+
+  - lat: -5.0
+    lon: -65.0
+    nz: ~  # all
+    atm: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.atmf006.nc"
+    sfc: "/scratch2/NCEPDEV/stmp3/Margaret.R.Marvin/global/COMROOT/Dust_FirstCase/gefs.20180816/00/mem000/model_data/atmos/history/gefs.t00z.sfcf006.nc"
+    out: "../tests/MetProfiles/Profile_AMAZON_Aug_06z.csv"
+
+variables:
+  # keys: MetState variables in lowercase
+
+  # Column coordinates
+  bxheight:  # layer thickness (m)
+    src: atm:delz
+  delp:  # pressure thickness (hPa)
+    src: atm:dpres
+    f: 100  # Pa -> hPa
+  pmid:
+    src: atm:pfull  # phalf are the interfaces
+  z:
+    src: atm  # combine hgtsfc (geopotential height at sfc) and delz (layer thickness; m)
+  zmid:
+    src: atm  # mid points of z
+
+  # Column State
+  cldf:  # cloud fraction
+    src: sfc:cldfra
+  ql:  # cloud liquid water mass fraction, but specifies units kg / kg dry air
+    src: atm:clwmr  # cloud liquid water mixing ratio
+  qv:  # specific humidity (kg/kg)
+    src: atm:spfh
+  sphu: # specific humidity (g/kg)
+    src: atm:spfh
+    f: 1000  # kg/kg -> g/kg
+  t:
+    src: atm:tmp
+  u:
+    src: atm:ugrd
+  v:
+    src: atm:vgrd
+
+  # Aerosol State
```

Comment:
With the way get-vars is currently working, I think we want to add `skip: true` to these, since they are not to be read into MetState, to skip them in the code generation.